### ISSUE TITLE
Link - UI issue with line mode

### ISSUE THREAD

**Issue by Julien-Dubois-eXo** [Meeds-io/meeds#1251]
**Steps to reproduce**

1. Go to a space
2. Add a link portlet with 4 links
3. Set it with the column mode

**(optional) Affected version**

Indicate the version where the bug is observed.

**Current behaviour**

There are several UI issues :  

- With big icons: there is too much white space between the portlet title and the image. Same there is too much space between the bottom of the text and the bottom of the portlet. It gives a feeling of something too big and not optimize.
- With small icons: The space below is better but there is still too much space above so the links are not aligned by the middle as there more space above
- Font size of the link in line (14px) is different from the font size in column (16px). I find it better in column. 

It's too bad because the column mode is very nice. Presently the line mode is take too much space on the stream. 
See video

https://github.com/Meeds-io/meeds/assets/105866381/17b20c93-b8b2-4851-a120-ed9dcd01f916

**Expected behavior**

Reduce the space to have something more harmonize with the column mode. 

**(optional) Affected environments**  
Please complete the following information:

**Comment by Julien-Dubois-eXo** [comment]
@srenault-meeds I guess it's more an enhancement but as it's more UI harmony issue I put it as a bug.

**Comment by srenault-meeds** [comment]
No a bug :) You request for an enhancement, so you have your answer.

**Comment by srenault-meeds** [comment]
It has been improved with an option (option to customize icons) I let you check

**Comment by Julien-Dubois-eXo** [comment]
I have tested it. We can do interesting things but it's not answering my need. My need is linked to margin management in the element.

**Comment by sirinebenabbes** [comment]
@Julien-Dubois-eXo  @srenault-meeds 

We need to reopen this Improvement feedback unless it will be also adressed by [2737  ](https://github.com/Meeds-io/meeds/issues/2737)  (let me know)

We still have the same issue on 7.x versions.

Besides te mentioned points, by @Julien-Dubois-eXo, if the links app is added in a small portlet,  there is a **useless** **scroll** displayed that gives a feeling of a not optimized display:

<img width="493" height="296" alt="Image" src="https://github.com/user-attachments/assets/f0485463-26e8-461f-ab0b-b470a514a70d" />


If we set a bigger margin, scroll won't be displayed yet there is too much white space between the portlet title and the icons.

Furthermore, as the margins required to correctly display the icons and text must be more than **220px**, in personalized display, choosing a lower value will only reduce the **lower part** without recentering the elements in the center of the block:

<img width="970" height="243" alt="Image" src="https://github.com/user-attachments/assets/33093742-5ee1-4a85-b31c-532e9dc9f79f" />


https://github.com/user-attachments/assets/31e71469-c945-4c3d-a62d-4505ebffdb2d


 
Related Ticket:
[81547](https://community.exoplatform.com/portal/dw/tasks/taskDetail/81547)

**Comment by srenault-meeds** [comment]
Do not open something again when this has been closed 11 months ago.
And about the scroll display, this is expected if you have fixed the height and the content cannot be displayed fully in the portlet's height.

**Comment by sirinebenabbes** [comment]
Ok noted @srenault-meeds

So do you mean that anyways even an enhancement request for this margin management in the element won't be considered, am I getting it right ?

**Comment by srenault-meeds** [comment]
No, this is is something different you are reporting. You are talking about a scroll display while fixing the height (which is expected) and Julien was talking about a margin problem within the link display (not the portlet) - reported in another feedback.

And if needed, create a new feedback issue

**Comment by sirinebenabbes** [comment]
ok thanks
